### DISCUSSION TITLE
Make Dialog Content bigger

### DISCUSSION THREAD

**Discussion by Max0u** [shadcn-ui/ui#1227] [Q&A] [answered]
Hello, is there a way to make the Dialog content bigger? I've tried to increase width on `DialogContent` but doesn't seems to affect anything

**Comment by shrix1** [comment, marked as answer]
Try changing the width in `dialog.tsx` from shadcn

**Reply by Max0u** [reply]
Yes, just needed to change the max width !

**Reply by adeludedperson** [reply]
![Screenshot 2023-11-04 at 9 24 28 PM](https://github.com/shadcn-ui/ui/assets/83860384/fa8b1342-6fc0-4238-af6a-0d8135a6af40)
